src/alexandria3k/data_sources/datacite.py

```diff
@@ -349,38 +328,18 @@ def Column(self, col):
         return super().Column(col)
 
 
-class SizesCursor(DataciteElementsCursor):
-    """A cursor over the work items' sizes data."""
-
-    def element_name(self):
-        """The work key from which to retrieve the elements. Not part of the
-        apsw API."""
-        return "sizes"
-
-    def Rowid(self):
-        """Return a unique id of the row along all records.
-        This allows for 128 sizes"""
-        return (self.parent_cursor.Rowid() << 7) | self.element_index
-
-    def Column(self, col):
-        """Return the value of the column with ordinal col"""
-        if col == 0:  # work_id
-            return self.parent_cursor.Rowid()
-        return super().Column(col)
-
-
-class FormatsCursor(DataciteElementsCursor):
-    """A cursor over the work items' formats data."""
+class RightsCursor(DataciteElementsCursor):
+    """A cursor over the work items' rights data."""
 
     def element_name(self):
         """The work key from which to retrieve the elements. Not part of the
         apsw API."""
-        return "formats"
+        return "rightsList"
 
     def Rowid(self):
         """Return a unique id of the row along all records.
-        This allows for 128 formats"""
-        return (self.parent_cursor.Rowid() << 7) | self.element_index
+        This allows for 1M rights"""
```

Comment:
I understand that `Rowid` is made up from the container index and the element index within the container.  Since there are 1000 elements in each container, a `<< 10` (1024) would be enough.  Use `<< 12` to provide a margin for error.